### PR TITLE
Javascript javascript2 week2

### PR DESCRIPTION
**Pull request by maccrin**
JS2-WEEK2-HOMEWORK

### CODE REVIEW

File: javascript/javascript2/week2/home-work.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+//doubling the odd numbers
+const myarray = new Array(22, 79, 34, 17, 1095, 678, 345, 982, 899, 6784, 72837, 56783, 98986);
```
Comment (naresh699) [inline javascript/javascript2/week2/home-work.js:2]: this code is perfectly fine, however, you also can use modern code like
myArray = [12,12,34,56]

File: javascript/javascript2/week2/hyfbayhelper/hyfBayHelper.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
+/* DONT MODIFY ANY OF THIS CODE!!!*/
+
+window.availableCountries = [
+    "Denmark",
+    "Sweden",
+    "Norway",
+    "Germany",
+    "Iceland",
+    "England",
```
Comment (naresh699) [inline javascript/javascript2/week2/hyfbayhelper/hyfBayHelper.js:9]: don't need , in last index

File: javascript/javascript2/week2/hyfbayhelper/main.js [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+console.log(products);
+const button = document.getElementById('click');
+const ul1 = document.getElementById('list');
+let filFlag = false;
+let result = [];
+let pageNo = 1;
+function createList(prods) {
+    const uid = document.getElementById('product');
+    uid.innerHTML = '';
+    for (const prod of prods) {
```
Comment (naresh699) [inline javascript/javascript2/week2/hyfbayhelper/main.js:11]: try writing es6 template string to write HTML.

**Review by naresh699 — APPROVED** [review]

File: javascript/javascript2/week2/home-work.js (hunk 1 above)
Comment (maccrin) [reply javascript/javascript2/week2/home-work.js:2]: 👍🏻

File: javascript/javascript2/week2/hyfbayhelper/main.js (hunk 3 above)
Comment (maccrin) [reply javascript/javascript2/week2/hyfbayhelper/main.js:11]: 👍🏻 sure,Thank You.

**Pull request merged**